RELEASE_NOTES.rst

```diff
@@ -1,3 +1,77 @@
+coala-bears 0.10.0
+==================
+
+This release, we had 97 commits contributed by 42 contributors
+
+We have updated the coala website to provide a quick overview of bears and
```

Comment:
not sure "We" is the correct pronoun. Shouldnt it be instead something like: "coala website has been updated to.." in release notes?